web/package/agama-web-ui.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Nov  6 06:06:51 UTC 2024 - Michal Filka <mfilka@suse.com>
+
+- url for downloading agama logs adapted to use new HTTP API
```

Comment:
```suggestion
- URL for downloading Agama logs adapted to use new HTTP API
```

Reply:
i tried but your eye is too sharp even in the morning ;-)